docs/docs/components/__tests__/expression-metadata.test.js

```diff
@@ -0,0 +1,24 @@
+import { types } from '../expression-metadata';
+import ref from '../../../../tsc/src/reference/latest';
+
+// help us catch that every type in expression-metadata.js exists in the style-spec
+// otherwise the site will not build
+describe('expression-metadata.js type exists in maplibre-gl-js-style-spec', () => {
```

Comment:
I'm not sure I understand what this test is actually testing.
It seems that this should be part of the regular tests of the style-spec and should not be here...